lgsm/functions/install_config.sh

```diff
@@ -261,6 +261,8 @@ elif [ "${gamename}" == "Double Action: Boogaloo" ]; then
 	fn_fetch_default_config
 	fn_default_config_remote
 	fn_set_config_vars
+elif [ "${gamename}" == "Factorio" ]; then
+	:
```

Comment:
Im updating this PR currently in this branch https://github.com/GameServerManagers/LinuxGSM/tree/feature/factorio